appinventor/components/src/com/google/appinventor/components/runtime/ReplForm.java

```diff
@@ -334,6 +324,24 @@ private void processExtrasAndData(Intent intent, boolean restart) {
     }
   }
 
+  private void startChromebook(String data, ReplForm me) {
```

Comment:
Because this is being called in the context of ReplForm, it seems to me that the `me` parameter is superfluous.

Reply:
In onCreateFinish it is called in the context of the OnInitializeListener instead of ReplForm.

-Jeff

Reply:
But since `onCreateFinish` doesn't exist in the `OnInitializeListener` interface, it will call the method in the outer context.

Reply:
i.e., the anonymous inner `OnInitializeListener` doesn't declare this method and isn't `static`, so `startChromebook` would resolve against the `ReplForm`'s `startChromebook` method.

Reply:
Oops. Right now that startChromebook is now part of ReplForm instead of being defined in-line... I can flush "me". I'll update this PR after I run my tests.